pkg/network/infraconfigurators/bridge.go

```diff
@@ -3,7 +3,6 @@ package infraconfigurators
 import (
 	"fmt"
 	"net"
-	"strconv"
```

Comment:
I like the commit, but, imo, the commit msg highlights the wrong thing (the mac address in bridge binding is a different sort of beast that needs special attention).

Please indicate that all link information - other than the mac address - can be recovered at phase2.

Reply:
done.

Reply:
typo in the commit msg, change from "Curretly, all ltje information," to "Currently, all the information".